CMakeLists.txt

```diff
@@ -682,3 +691,12 @@ if(BUILD_EXAMPLE_APPLICATIONS)
     add_subdirectory(onboarding_tool)
     add_subdirectory(apps)
 endif()
+
+if(MINGW)
+    execute_process(
+        COMMAND touch test.c
+        COMMAND gcc -dM -E test.c
+        OUTPUT_VARIABLE output
+    )
+    message(STATUS ${output})
+endif()
```

Comment:
What is this for?

If it is only for some debugging, please don't forget to remove it before merging.

Reply:
Removed, I wanted to know the problematic MinGW GCC version.